Hello_Object_Oriented_Due_Dec_1/renderBoard.js

```diff
@@ -0,0 +1,132 @@
+
+
+class Task {
+  constructor(taskName='', assignee='') {
+    this.taskName = taskName;
+    this.assignee = assignee;
+  }
+
+  render() {
+    if (this.assignee != '' && this.taskName != '') {
+
+      return `${this.taskName} ${this.assignee}`;
+    }
+    else if (this.taskName != '') {
+
+      return `${this.taskName}`;
+    } else {
+      return '';
+    }
+  }
+}
+
+
+class List {
+
+  constructor(listName) {
+    this.listName = listName;
+    this.lists = []; // store tasks
+  }
+  addTask (task) {
+    this.lists.push(task);
```

Comment:
you should check the type of `taskObj` to make sure it is an `instanceof` `Task`
This guarantees that even if something other than a task were passed as an argument to this method, the `addTask` method knows not to add it to its `listHasTasks` array
```javascript
if (taskObj instaceof Task) {
//...
}
```
Otherwise, your `addTask` method could push more than only tasks into the `listHasTasks` array
```javascript
console.log(this.listHasTasks) // []
list.addTask('this is not a task but a string');
console.log(this.listHasTasks) // ['this is not a task but a string']
```